doc/sphinx/practical-tools/utilities.rst

```diff
@@ -130,7 +130,11 @@ Note: building with `dune` is experimental.  See :ref:`building_dune`.
 The `_CoqProject` file contains the information needed to generate a makefile
 for building your project.  Your `_CoqProject` file should be in
 the top directory of your project's source tree.  We recommend using the
-:term:`logical name` of the project as the name of the top directory.
+:term:`logical name` of the project as the name of the top directory.  (Make
```

Comment:
Would it be better to make this a note? It is rather a separate concern from the rest of the material.

Reply:
How about if I make it a separate paragraph and start it with **Note:**?  I think our current note styling in most cases takes too much space and draws too much attention; it's a bit disruptive:

![image](https://user-images.githubusercontent.com/1253341/205187510-07d6d12a-86d1-4e85-8c29-21e13dcb81c9.png)

Reply:
I think it is OK to draw attention, that means it can easily be ignored too. Having a long block of text will not help anybody skimming.

Reply:
I said I'd make it a separate paragraph.  The text won't be any shorter if it's in a `.. note::`--and it will take up more space on the screen if it is.

I think reviewers of PRs should accept that the submitters may choose to do things somewhat differently that the reviewer would and that many of those differences don't matter.  Though, as social creatures, we do love discussing things.

I'm willing to address some very small suggestions immediately if only because writing a PR is overkill (maybe more work that just fixing it) and otherwise the suggestion will probably be lost.  Let's keep it simple.  If these very small suggestions start taking significant time, I will ignore more of them.

Check out my change.

Reply:
PS It will also look similar to the Note just a few lines above:  "Note: building with dune is experimental."